collectors/cmdb/hpsm/src/main/java/com/capitalone/dashboard/collector/DefaultHpsmClient.java

```diff
@@ -779,6 +780,10 @@ private void handleChangeSoapMessage(SOAPBodyElement keysTag) throws SOAPExcepti
 		incident.setIncidentID(getStringValueFromMap(map,HpsmCollectorConstants.INCIDENT_ID));
 		incident.setCategory(getStringValueFromMap(map,HpsmCollectorConstants.INCIDENT_CATEGORY));
 		incident.setOpenTime(getStringValueFromMap(map,HpsmCollectorConstants.INCIDENT_OPEN_TIME));
+		String closedTime = getStringValueFromMap(map,HpsmCollectorConstants.INCIDENT_CLOSE_TIME);
+		if (!StringUtils.isEmpty(closedTime)) {
```

Comment:
Why do we need StringUtils here?

Reply:
I am just using StringUtils isEmpty method to check if the string is empty. In certain cases "closedTime" turns out to be empty.

Reply:
@subodhbattina Since you already retrieved closedTime in line 783 why cant that be set in line 785.

Reply:
@rvema Good catch thanks, this was by oversight, have fixed it.